From 7471359617603030ac753aef571ebe2e7299a86c Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 26 Jun 2016 10:50:42 +0300
Subject: [PATCH 12/24] ARM: i.MX6: dts: sata refactoring

1) enable "target-supply" regulator.
2) add FreeScale gpr13 values for sata_phy tunning.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/boot/dts/imx6q-cm-fx6.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6q-cm-fx6.dtsi b/arch/arm/boot/dts/imx6q-cm-fx6.dtsi
index 3a10e5e..0cdba81 100644
--- a/arch/arm/boot/dts/imx6q-cm-fx6.dtsi
+++ b/arch/arm/boot/dts/imx6q-cm-fx6.dtsi
@@ -91,5 +91,10 @@
 
 /* sata */
 &sata {
+	target-supply = <&reg_sata_nstandby2>;
 	status = "okay";
+	fsl,transmit-level-mV = <1104>;
+	fsl,transmit-boost-mdB = <370>;
+	fsl,transmit-atten-16ths = <9>;
+	fsl,receive-eq-mdB = <3000>;
 };
-- 
1.9.1

